trade-finance-manager-ui/templates/utilisation-reports/_macros/premium-payments-table.njk

```diff
@@ -3,8 +3,109 @@
 {% import "../../_macros/table-cell-checkbox.njk" as tableCellCheckbox %}
 {% import "../../_macros/select-all-table-cell-checkbox.njk" as selectAllTableCellCheckbox %}
 
+{% macro renderPremiumPaymentsTableRow(params) %}
+  {% set feeRecordPaymentGroup = params.feeRecordPaymentGroup %}
+
+  {% set feeRecords = feeRecordPaymentGroup.feeRecords %}
+  {% set totalReportedPayments = feeRecordPaymentGroup.totalReportedPayments %}
+  {% set paymentsReceived = feeRecordPaymentGroup.paymentsReceived %}
+  {% set totalPaymentsReceived = feeRecordPaymentGroup.totalPaymentsReceived %}
+  {% set status = feeRecordPaymentGroup.status %}
+  {% set displayStatus = feeRecordPaymentGroup.displayStatus %}
+  {% set checkboxId = feeRecordPaymentGroup.checkboxId %}
+  {% set isChecked = feeRecordPaymentGroup.isChecked %}
+
+  {% for feeRecord in feeRecords %}
+    {% set isFirstRow = loop.index === 1 %}
+    {% set isLastRow = loop.index === feeRecords.length %}
+
+    <tr
+      class="govuk-table__row"
+      data-cy="premium-payments-table-row--feeRecordId-{{ feeRecord.id }}"
+    >
+      <th
+        scope="row"
+        class="govuk-table__header govuk-!-font-weight-regular{{ " no-border" if not isLastRow }}"
+      >
+        {{ feeRecord.facilityId }}
+      </th>
+      <td class="govuk-table__cell{{ " no-border" if not isLastRow }}">
+        {{ feeRecord.exporter }}
+      </td>
+      <td class="govuk-table__cell govuk-table__cell--numeric{{ " no-border" if not isLastRow }}">
+        {{ feeRecord.reportedFees }}
+      </td>
+      <td class="govuk-table__cell govuk-table__cell--numeric{{ " no-border" if not isLastRow }}">
+        {{ feeRecord.reportedPayments }}
+      </td>
+
+      {% if isFirstRow %}
+        <td
+          class="govuk-table__cell govuk-table__cell--numeric{{ " no-border" if not isLastRow }}"
+          data-sort-value="{{ totalReportedPayments.dataSortValue }}"
+        >
+          {{ totalReportedPayments.formattedCurrencyAndAmount }}
+        </td>
+      {% elif isLastRow %}
+        <td class="govuk-table__cell" data-sort-value="{{ totalReportedPayments.dataSortValue }}"></td>
+      {% else %}
+        <td class="govuk-table__cell no-border" data-sort-value="{{ totalReportedPayments.dataSortValue }}"></td>
+      {% endif %}
```

Comment:
Could we replaced this if else block with the following?
```
        <td
          class="govuk-table__cell govuk-table__cell--numeric{{ " no-border" if not isLastRow }}"
          data-sort-value="{{ totalReportedPayments.dataSortValue }}"
        >
          {{ totalReportedPayments.formattedCurrencyAndAmount if isFirstRow else "" }}
        </td>
```

Reply:
That's a really nice suggestion! I had stared at this for so long that I was struggling to come up with a way to make it more succinct, so thank you :)